src/mpid/ch4/netmod/ucx/ucx_recv.h

```diff
@@ -49,6 +49,35 @@ static inline void MPIDI_UCX_recv_cmpl_cb(void *request, ucs_status_t status,
 }
 
 #undef FUNCNAME
+#define FUNCNAME MPIDI_UCX_mrecv_cmpl_cb
+#undef FCNAME
+#define FCNAME MPL_QUOTE(FUNCNAME)
+static inline void MPIDI_UCX_mrecv_cmpl_cb(void *request, ucs_status_t status,
+                                           ucp_tag_recv_info_t * info)
+{
+    MPIDI_UCX_ucp_request_t *ucp_request = (MPIDI_UCX_ucp_request_t *) request;
+
+    MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDI_UCX_MRECV_CMPL_CB);
+    MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_UCX_MRECV_CMPL_CB);
+
```

Comment:
We maybe have to add error-checking here. Since mrecv also as a "count" parameter, the message can be truncated. 

This also has to be changed in the mrecv function